Monika-After-Story/MonikaModDev
# Adjusted topics based on starting act

Adjusted topics based on starting act, so they also apply to those that never deleted Monika and went past act 3.

@jmwall24 here - Alright, so this got bigger than originally intended. Major changes include `monika_credits_song` is now a one-time-only topic that depends on the player's starting act. Players with `monika_kill` as True will get the YR path, while people with it False will get the new Our Reality path. In addition to this, Monika now has the ability to actually play both songs for you. We've added a new piano sprite (thanks @Orcaramelo!) and for YR, we see Monika behind her piano singing the song as she plays (using the credits song so you actually hear her voice). For OR, same use of the piano, but instead she 'sings' the song via on-screen lyrics. After this topic plays, everyone will have YR unlocked to view in this manner, while only people with `monika_kill` as False will have OR unlocked. There also have been some minor edits and additions to transforms to give us more control of slides (direction and speed).

## Testing

- upon load, run the update script for `v0_10_8` and verify that `monika_credits_song` is no longer random or unlocked, and that it now has the appropriate conditional and action.

- verify if you meet the conditional, the topic is queued and that you get the correct path based on your starting act (you can edit the appropriate vars to test both paths)

- verify the songs play correctly and all shields etc are properly enabled and disabled

- verify after the topic is complete, the appropriate song event(s) is/are unlocked
